src/redux/apiReducersList.ts

```diff
@@ -7,3 +7,5 @@ export const apiReducersList = {
 } as const
 
 export const apiMiddlewares = [transactionFeedV2Api.middleware]
+
+export const apiReducersKeys = Object.keys(apiReducersList)
```

Comment:
@jeanregisser Not really, just thought it might be a little more convenient to have api-related stuff exported from this file but I am totally fine with both approaches. I'll revert this one.

Reply:
We can keep it I guess, I was just wondering why it needed to change now.